rbac/management/group/view.py

```diff
@@ -382,9 +382,19 @@ def remove_principals(self, group, principals, account=None, org_id=None):
 
             for username in principals:
                 try:
-                    principal = Principal.objects.get(username__iexact=username, tenant=tenant)
+                    principal = Principal.objects.get(username__iexact=username, tenant=tenant, group=group)
```

Comment:
My suggestion is here to check existence of all usernames in DB before we process them:
- when all of them exist
  - then we should process all 
- otherwise return err and don't process any of them.